src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -1246,8 +1255,21 @@ class DotVVM {
                     element.style.display = "none";
                 }
 
-                dotvvm.isPostbackRunning.subscribe(e => {
-                    if (e) {
+                dotvvm.updateProgressChangeCounter.subscribe(e => {
+                    let shouldRun = false;
+
+                    if (includedQueues.length === 0) {
+                        for (let queue in dotvvm.postbackQueues) {
+                            if (excludedQueues.indexOf(queue) < 0 && dotvvm.postbackQueues[queue].noRunning) {
+                                shouldRun = true;
+                                break;
+                            }
+                        }
+                    } else {
+                        shouldRun = includedQueues.some(q => dotvvm.postbackQueues[q] && dotvvm.postbackQueues[q].noRunning);
```

Comment:
This condition (and the complementary one above too) seems strange, I think the `noRunning` means something different from "nothing is running in this queue". However, I can't look for context on my phone, so maybe I just don't get something... will have a look at that later.

Reply:
`noRunning` is number of running postbacks in that queue. I'll add `> 0` so it is more clear.